src/bin/add/args.rs

```diff
@@ -141,13 +141,42 @@ pub struct Args {
     /// Registry to use
     #[structopt(long = "registry", conflicts_with = "git", conflicts_with = "path")]
     pub registry: Option<String>,
+
+    /// print available features inline. Useful for testing
+    #[structopt(long = "inline")]
+    pub inline: bool,
```

Comment:
I'm unsure of the value of this for end users.  Adding extra stuff to the UI can make it harder for people to find what they want.

What is the challenge with testing with out it?  I'm wondering if we can find an alternative.

Reply:
That's fair. The challenge is that when you print things multiline the testing framework only captures the first line. so for example the test `lists_features_path_dependency` wouldn't actually be able to capture the listed features. I guess we could rely on `CARGO_IS_TEST` being set in the env?

Reply:
>  the testing framework only captures the first line.

This is surprising to me.  What would be reproduction steps if I were to dig into this?

Reply:
I went back to try it and actually what I said turns out to not be true the framework does capture it correctly. It must have been caused by another error on my part earlier that has since been resolved. i'll remove the inline code.